api/src/main/java/org/apache/xtable/model/storage/OneFileGroup.java

```diff
@@ -28,20 +28,20 @@
 
 import org.apache.xtable.model.stat.PartitionValue;
 
-/** Represents a grouping of {@link OneDataFile} with the same partition values. */
+/** Represents a grouping of {@link InternalDataFile} with the same partition values. */
 @Value
 @Builder
 public class OneFileGroup {
```

Comment:
Should we update this as well to InternalFileGroup? Hudi has a concept of file group as well. Or maybe there is a better name for this since it is representing a list of files within a single partition.

Reply:
Yes, it should be better to fix prefix of this file as well. 
BTW, #393 is the umbrella issues for all prefix renaming/removing tasks. I can incorporate this class in its task as well. To make PR reviews easy, my plan is to create multiple PRs.

Reply:
ok sounds good to me